site/docs/v1/tech/apis/form-fields/_form-field-key.adoc

```diff
@@ -6,15 +6,22 @@ The key is the path to the value in the `user` or `registration` object.
 +
 Predefined keys are typed values managed by FusionAuth. When using a pre-defined key, the field data type and form control are not configurable.
 +
+ * `registration.preferredLanguages`
+ * `registration.roles`
+ * `registration.timezone`
  * `registration.username`
  * `user.birthDate`
  * `user.email`
  * `user.firstname`
  * `user.fullName`
+ * `user.imageUrl`
  * `user.lastName`
  * `user.middleName`
  * `user.mobilePhone`
  * `user.password`
+ * `user.preferredLanguages`
+ * `user.timezone`
+ * `user.twoFactorEnabled`
```

Comment:
These new ones will need a since flag since you won't be able to create them prior to 1.20.0